packages/storage-plus/src/keys.rs

```diff
@@ -9,6 +10,9 @@ pub trait PrimaryKey<'a>: Clone {
     type Prefix: Prefixer<'a>;
     type SubPrefix: Prefixer<'a>;
 
+    type Suffix: KeyDeserialize;
+    type SuperSuffix: KeyDeserialize;
```

Comment:
What are `Suffix` and `SuperSuffix`? It requires really going deep to the code and it could not if there would be solid doc coment here - we need to start using doc coments as they makes live far easier - it works with `cargo doc`, it works with ides, and it helps understanding read code.

Reply:
Good point. I agree. I know your questions is kind of rethoric, but `SuperSuffix` is the complement of `SubPrefix`, in the same way `Suffix` is the complement of `Prefix`.

Reply:
No, they were not rethoric at all. At the point of writing this comment I was really not sure about that, and it got mi confused. I understood it way later. I am not ironical, at least not on review - I like have as clear conversation as it is possible.